pkg/snapstore/swift_snapstore_test.go

```diff
@@ -150,3 +178,50 @@ func handleDeleteObject(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusNotFound)
 	}
 }
+
+// handleBulkDeleteObject creates an HTTP handler at `/testContainer/testObject` on the test handler mux that
+// responds with a `Delete` response.
+func handleBulkDeleteObject(w http.ResponseWriter, r *http.Request) {
+	objectMapMutex.Lock()
+	defer objectMapMutex.Unlock()
+
+	var jsonResponse bulkDeleteJSONResponse
+
+	buf := new(bytes.Buffer)
+	if _, err := io.Copy(buf, r.Body); err != nil {
+		errorMessage := fmt.Sprintf("failed to read content %v", err)
+		logrus.Errorf(errorMessage)
+		jsonResponse.ErrorStrings = append(jsonResponse.ErrorStrings, errorMessage)
+		w.WriteHeader(http.StatusOK)
+		return
+	}
+
+	segmentObjects := strings.Split(string(buf.Bytes()), "\n")
+	// last string is an empty string, artefact of '\n' at the end
+	segmentObjects = segmentObjects[:len(segmentObjects)-1]
+	for _, segmentObject := range segmentObjects {
+		segmentObject = "/" + segmentObject
+		// objects.BulkDelete() internally calls url.QueryEscape
+		unescapedQueryForObject, err := url.QueryUnescape(segmentObject)
+		if err != nil {
+			errorMessage := fmt.Sprintf("failed to unescape url %v", err)
+			logrus.Errorf(errorMessage)
```

Comment:
```suggestion
			logrus.Errorf(fmt.Sprintf("failed to unescape url %v", err))
```
Saves a line. Please follow this convention for other occurrences as well.

Reply:
Understood, will changes all occurrences to follow this.